### PR TITLE
Add support for  MWC address encoding and decoding functionality

### LINKED ISSUE #208: Add support for MimbleWimbleCoin (MWC) addresses in encoding library
Here is the encoding library: https://github.com/ensdomains/address-encoder

Example: https://github.com/ensdomains/address-encoder/pull/47/files

Discord for dev help: https://discord.com/invite/AskZbFx

Please provide a reference link so we can check you did the encoding correctly.

We'd like to keep the encoding library as small as possible, so please be efficient in your use of space.

We will likely be reviewing PRs and paying out bounties more or less on a weekly basis. No need to ping us to check your PR, we will definitely get to it.

Thanks for your help!

### PR DESCRIPTION
**Pull request by janus**
## Issue number
closes #208 

## Description
Connect MWC to used bs58Encode and bs58Decode

## List of features added/changed
Not much coding only specifying MWC to used bs58Encode and bs58Decode

## How Has This Been Tested?
Add MWC in the test pool so that decoded address should match it text version

## Checklist:

- [X ] My code follows the code style of this project.
- [ X] My code implements all the required features.

### CODE REVIEW

File: src/__tests__/index.test.ts [hunk 1]
```diff
@@ -363,6 +363,11 @@ const vectors: Array<TestVector> = [
       { text: 'AGmV3oHqzfAs3VFiqmn6cecxCXVNyg6tNh', hex: '0ae542fee226c044dc19b036db7cec939777596f' },
     ],
   },
+  {
+    name: 'MWC',
+    coinType: 1088,
```
Comment (makoto) [inline src/__tests__/index.test.ts:368]: Where did you get the coinType? MWC(aka MimbleWimbleCoin) does not seem to be listed on SLIP 44. 

https://github.com/satoshilabs/slips/blob/master/slip-0044.md

**Review by makoto — CHANGES_REQUESTED** [review]
https://github.com/ensdomains/address-encoder/pull/234#pullrequestreview-516683471

**Comment by janus** [comment]
https://github.com/satoshilabs/slips/blob/master/slip-0044.md
MWC or Mimblewimble not in the list

**Comment by makoto** [comment]
Closing due to lack of SLIP44 number

**Pull request closed without merging**